lib/vector-core/src/event/vrl_target.rs

```diff
@@ -81,7 +87,7 @@ impl Iterator for TargetIter<TraceEvent> {
 }
 
 impl VrlTarget {
-    pub fn new(event: Event, info: &ProgramInfo) -> Self {
+    pub fn new(event: Event, info: &ProgramInfo, multi_value_metric_tags: bool) -> Self {
```

Comment:
Could this parameter go into `ProgramInfo` instead?

Reply:
`ProgramInfo` is info generated by the compiler. The compiler is unable (and shouldn't be able) to decide how to handle tags. That's something the user decides.